src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,17 @@
+.TodoInfo {
+  padding: 8px;
```

Comment:
Consider using more descriptive class names instead of `TodoInfo`. For example, `TaskDetails` or `TaskInfo`. 🤔